Backup (dump) command fails with stacktrace - latest tag on dockerhub (1.7.6 works fine)

- Gitea version (or commit ref): Gitea version caba282 built with GNU Make 4.2.1, go1.12.4 : bindata, sqlite, sqlite_unlock_notify
- Git version: NA
- Operating system: docker image gitea/gitea:latest
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description

When trying to run the backup I get the following stack trace:
```
git-67778cc5ff-xmz9c:/app/gitea$ ./gitea dump -c /data/gitea/conf/app.ini
dump flag redefined: v
panic: dump flag redefined: v

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc000168780, 0x1c4ea00, 0xc000040cf5, 0x199833c, 0x1, 0x19a9963, 0x11)
	/usr/local/go/src/flag/flag.go:850 +0x4af
flag.(*FlagSet).BoolVar(...)
	/usr/local/go/src/flag/flag.go:623
flag.(*FlagSet).Bool(0xc000168780, 0x199833c, 0x1, 0x0, 0x19a9963, 0x11, 0x1)
	/usr/local/go/src/flag/flag.go:636 +0x8b
github.com/urfave/cli.BoolFlag.Apply.func1(0x199833c, 0x1)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/flag.go:274 +0xb3
github.com/urfave/cli.eachName(0x1998333, 0xa, 0xc0005f1770)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/flag.go:78 +0xdc
github.com/urfave/cli.BoolFlag.Apply(0x1998333, 0xa, 0x19a9963, 0x11, 0x0, 0x0, 0x0, 0x0, 0xc000168780)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/flag.go:269 +0xfa
github.com/urfave/cli.flagSet(0x198a6e5, 0x4, 0xc0002f2960, 0x9, 0xa, 0x1)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/flag.go:69 +0x90
github.com/urfave/cli.Command.Run(0x198a6e5, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x19c5241, 0x1d, 0x0, ...)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/command.go:94 +0xc1
github.com/urfave/cli.(*App).Run(0xc0002a8820, 0xc00003c100, 0x4, 0x4, 0x0, 0x0)
	/go/src/code.gitea.io/gitea/vendor/github.com/urfave/cli/app.go:241 +0x658
main.main()
	/go/src/code.gitea.io/gitea/main.go:100 +0x75b
git-67778cc5ff-xmz9c:/app/gitea$ 
```

...


## Screenshots

NA

--- a/cmd/dump.go
+++ b/cmd/dump.go
@@ -36,7 +36,7 @@ It can be used for backup and capture Gitea server image to send to maintainer`,
 			Usage: "Name of the dump file which will be created.",
 		},
 		cli.BoolFlag{
-			Name:  "verbose, v",
+			Name:  "verbose, V",
 			Usage: "Show process details",
 		},
 		cli.StringFlag{

--- a/docs/content/doc/usage/command-line.en-us.md
+++ b/docs/content/doc/usage/command-line.en-us.md
@@ -147,7 +147,7 @@ in the current directory.
     - `--tempdir path`, `-t path`: Path to the temporary directory used. Optional. (default: /tmp).
     - `--skip-repository`, `-R`: Skip the repository dumping. Optional.
     - `--database`, `-d`: Specify the database SQL syntax. Optional.
-    - `--verbose`, `-v`: If provided, shows additional details. Optional.
+    - `--verbose`, `-V`: If provided, shows additional details. Optional.
 - Examples:
     - `gitea dump`
     - `gitea dump --verbose`

